e2e/testdata/simple-bundle.golden

```diff
@@ -0,0 +1,96 @@
+{
+	"name": "acmecorp/simple",
+	"version": "1.1.0-beta1",
+	"description": "new fancy webapp with microservices",
+	"maintainers": [
+		{
+			"name": "John Developer",
+			"email": "john.dev@acmecorp.cool",
+			"url": ""
+		},
+		{
+			"name": "Jane Developer",
+			"email": "jane.dev@acmecorp.cool",
+			"url": ""
+		}
+	],
+	"invocationImages": [
+		{
+			"imageType": "docker",
+			"image": "acmecorp/simple:1.1.0-beta1-invoc"
+		}
+	],
+	"images": null,
+	"actions": {
+		"inspect": {
+			"Modifies": false
+		}
+	},
+	"parameters": {
+		"api_host": {
+			"type": "string",
+			"defaultValue": "coolapp.com",
+			"required": false,
+			"metadata": {},
+			"destination": {
+				"path": "",
+				"env": "docker_param1"
+			}
+		},
+		"docker.kubernetes-namespace": {
+			"type": "string",
+			"defaultValue": "",
+			"required": false,
+			"metadata": {
+				"description": "Namespace in which to deploy"
+			},
+			"destination": {
+				"path": "",
+				"env": "DOCKER_KUBERNETES_NAMESPACE"
+			}
+		},
+		"docker.orchestrator": {
+			"type": "string",
+			"defaultValue": "",
+			"allowedValues": [
+				"",
+				"swarm",
+				"kubernetes"
+			],
+			"required": false,
+			"metadata": {
+				"description": "Orchestrator on which to deploy"
+			},
+			"destination": {
+				"path": "",
+				"env": "DOCKER_STACK_ORCHESTRATOR"
+			}
+		},
+		"static_subdir": {
+			"type": "string",
+			"defaultValue": "data/static",
+			"required": false,
+			"metadata": {},
+			"destination": {
+				"path": "",
+				"env": "docker_param2"
+			}
+		},
+		"web_port": {
+			"type": "string",
+			"defaultValue": "8082",
+			"required": false,
+			"metadata": {},
+			"destination": {
+				"path": "",
+				"env": "docker_param3"
+			}
+		}
+	},
+	"credentials": {
+		"docker.context": {
+			"path": "/cnab/app/context.dockercontext",
+			"env": ""
+		}
+	}
+}
```

Comment:
nit: missing an empty line

Reply:
Hmmm... The bundle file doesn't have it neither...
Should we include it?

Reply:
The golden should match the output so this might be a configuration issue with the marshaller that marshals the bundle.json.

Reply:
Actually the marshaller has no such configuration. What is done in the case of `render --formatter json` is that a `\n` is added by hand:
https://github.com/docker/app/blob/3603491fdcc72ebf31195a5ba092e50628dbff38/internal/formatter/json/driver.go#L24

Reply:
Well, for now, as the `docker-app bundle` command does not include a `\n`, I will let it like this.
In the future, if we want to add a `\n` in the `bundle`, then the golden file should adapt to it.